app/src/main/res/layout/activity_gallery.xml

```diff
@@ -3,10 +3,19 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:background="@drawable/background"
     tools:context=".SettingsActivity">
 
     <!-- Toolbar -->
     <include layout="@layout/toolbar"/>
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/gallery_recyclerview"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingTop="80dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/toolbar"/>
```

Comment:
Place the toolbar after the RecyclerView to make sure that the recyclerView does not overdraw the toolbar and makes the back button not usable. This makes the test TestToolbarBackButton fail.
```suggestion
</androidx.constraintlayout.widget.ConstraintLayout> 
    <androidx.recyclerview.widget.RecyclerView
        android:id="@+id/gallery_recyclerview"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:paddingTop="80dp"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toBottomOf="@+id/toolbar"/>
        
        <!-- Toolbar -->
        <include layout="@layout/toolbar"/>
```